src/custom-types.json

```diff
@@ -236,6 +236,14 @@
   "application/x-7z-compressed": {
     "compressible": false
   },
+  "application/x-arj": {
+    "compressible": false,
+    "extensions": ["arj"],
+    "sources": [
+      "https://en.wikipedia.org/wiki/List_of_archive_formats",
+      "http://www.wussu.com/various/mimetype.htm"
```

Comment:
Unfortunately I may have to give up on finding an official source on this. No emails back, no responses on FTN networks with good info, etc. 

Where does that put us?